Mac/Resources/framework/Info.plist.in

```diff
@@ -24,6 +24,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>%VERSION%</string>
+	<key>MinimumOSVersion</key>
+	<string>@MACOSX_DEPLOYMENT_TARGET@</string>
```

Comment:
From the Apple developer docs, it appears that `MinimumOSVersion` does not apply to macOS; there is a similar `LSMinimumSystemVersion` that [is documented](https://developer.apple.com/documentation/bundleresources/information-property-list/lsminimumsystemversion) as applying to Mac App Store submissions. Is there a particular reason for adding either here? It seems out of place.

Reply:
This is a stray piece of an attempt to generate an XCFramework for macOS as well; I'll remove it from the PR as it's unrelated, but I'll also make a note that the key needs to be `LSMinimumSystemVersion` on macOS - I'd missed that the key was different to other Apple platforms.